modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -20,13 +25,18 @@ const RTCIceGatheringState = {
     complete: 'complete'
 };
 
+const CNAME = `jitsi-ortc-cname-${RandomUtil.randomInt(10000, 99999)}`;
```

Comment:
Ummmm, actually CNAME should be the same for all the PC created by the same physical endpoint.